### ISSUE TITLE
Single handed Scaling

### ISSUE THREAD

**Issue by RueShejn** [Yellow-Dog-Man/Resonite-Issues#6416]
### Is your feature request related to a problem? Please describe.

for accessibility there should be an option for scaling using a single hand/controller to accommodate those with missing or low functioning limbs
 this should function for users avatar and objects

### Describe the solution you'd like

scaling via a context menu slider

### Describe alternatives you've considered

n/a

### Requesters

Rue Shejn, Ryuvi, Aegis_Wolf

**Comment by mxjessie** [comment]
Dupe of #540, related to #2329

**Comment by TisFoolish** [comment]
As one of these users I'd like more than a context menu aid they're slow, cumbersome, inaccurate, and less  immersive than a physical gesture. 

My personal solution is an invisible slider on my arm i can grab with my working hand to scale myself up or down. 

Another, similar set up would be too have it on the 'chest'. Then you slide it up to scale yourself up and down to scale down.

**Comment by Frooxius** [comment]
Closing as duplicate of #540
